Retrospectives: Initial description (#27)
wegotpop/principles-and-practices: practices/retrospectives/functional-retrospective.md

---
layout: page
title: Functional Retrospective
---

This is a recurring (currently once every two weeks) retrospective that due to historical reasons is also sometimes called the "Sprint Retrospective".

## Format

This is typically a simple "Went Well", "Didn't go well" format with topics being written anonymously then grouped by a facilitator before being voted on in terms of priority to discuss.

The retrospective is typically timeboxed and also includes a review of previous action items.

## What went well?

The functional retrospective is meant to capture individual sentiment on issues and is meant to provoke a discussion that helps align and normalise views across the group.

Only raise issues that you personally understand and believe in. If you think other people have problems that they are not raising at the retro you should raise and issue *about that* rather than trying to add an issue that you might not fully understand.

Whether something went well or not is entirely based on how you feel it about. The items are grouped by theme so it's not uncommon to have the same topic containing elements that are positive and negative.

> There is also some wriggle room, typically new starters get a "Went Well" topic even though there isn't anything to discuss and there may have been problems during the onboarding process.

Reviewer: Maybe
```suggestion
There is also some wriggle room, typically new starters get a "Went Well" topic even though there isn't anything to discuss or there have been problems during the onboarding process.
```
